test/verify/check-system-info

```diff
@@ -380,5 +380,43 @@ class TestSystemInfo(MachineCase):
         b.enter_page("/system")
 
 
+    @skipImage("PCP config not yet in manifest and no packagekit", "rhel-7-5")
+    def testPcpAvailable(self):
+        m = self.machine
+        b = self.browser
+
+        self.login_and_go("/system")
+
+        # images which have PCP (most of them) should not show the "enable pcp" link
+        if not m.atomic_image and m.image not in ["debian-stable"]:
+            b.wait_visible("#server-pmlogger-onoff-row")
+            b.wait_not_visible("#system-information-enable-pcp")
+
+            # remove the pcp package
+            if m.image.startswith("debian") or m.image.startswith("ubuntu"):
+                m.execute("apt-get remove -y pcp")
+            elif m.image.startswith("fedora") or m.image in ["rhel-x"]:
+                m.execute("dnf remove -y pcp")
+            elif m.image.startswith("centos") or m.image.startswith("rhel-7"):
+                m.execute("pkcon remove -y pcp")
+            elif m.image.startswith("rhel-7"):
+                m.execute("yum remove -y pcp")
+            else:
+                raise NotImplementedError("unknown image " + m.image)
```

Comment:
Couldn't this always use `pkcon remove` on all OSes?

Reply:
Of course. Perfect.